pages/dkp/kommander/1.3/projects/platform-services/platform-services-catalog/kudo-zookeeper/index.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: layout.pug
+navigationTitle: KUDO Zookeeper
+render: mustache
+title: KUDO Zookeeper
+menuWeight: 3
+excerpt: Day 2 Operations of Apache Zookeeper
+beta: false
+---
+
+<!-- markdownlint-disable MD018 -->
+## KUDO Zookeeper
+
+Kommander Catalog adds integration for [KUDO Zookeeper Operator](https://github.com/kudobuilder/operators/tree/master/repository/zookeeper), which simplifies day 2 operations of [Apache Zookeeper](https://zookeeper.apache.org/). 
+
+#include /dkp/kommander/1.3/include/kudo-intro.tmpl
+
+It is **strongly recommended** to view the [KUDO Zookeeper Documentation](https://github.com/kudobuilder/operators/tree/master/repository/zookeeper/docs/latest) which covers the KUDO Zookeeper Operator in-depth. This document covers the integration aspects of KUDO Zookeeper Operator with D2iQ Kommander.
+
+### Kommander Catalog
+
+KUDO Zookeeper is located in the Kommander Catalog.
+#include /dkp/kommander/1.3/include/kommander-catalog-drilldown.tmpl
+
+### Installation
+From the [Project Catalog](/dkp/kommander/1.3/projects/platform-services/) select the desired version of Zookeeper and click Deploy.
+
+Here is an example of what should appear in the Kommander UI, the dialog is populated with sensible defaults:
+
+![Zookeeper Service Install Configuration](/dkp/kommander/1.3/img/platform-services-zookeeper-config-dialog.png)
+
+- The **ID** field above is referred to as the Zookeeper `instance` by KUDO.
```

Comment:
Not sure if this is really an in-scope comment on this doc, but when I last played with this stuff, I renamed the zookeeper instance on install but then I was super confused as to what the Zookeeper URI was.   You need to know this, of course, for other services. 

It would be nice if this were documented better in the external access section below, as there is no mention of the service names there, and a reference to the external access section added here to tie the two together?  (Something like `The instance is part of the the external name of the service; see the external access section for more details?`)

Reply:
@cneth I've now added docs to make this clearer for both KUDO ZK and also Kafka in the associated PR in #3532

Reply:
Hi @kaiwalyajoshi - thanks, that's a big improvement!